src/app/connections/list-toolbar/list-toolbar.component.ts

```diff
@@ -91,4 +109,12 @@ export class ConnectionsListToolbarComponent<T> implements OnInit, OnDestroy {
     });
     this.filteredItems.next(result);
   }
+
+  filterFieldSelected($event: FilterEvent) {
+    const field = $event.field;
+    if (field.id === 'tag') {
+      field.queries = _.uniq(_.flatMap(this._allItems, item => item['tags'] || []))
```

Comment:
hmm, might be a bit trickier without lodash, but:

flatMap => https://stackoverflow.com/questions/39837678/why-no-array-prototype-flatmap-in-javascript
uniq => https://stackoverflow.com/questions/11246758/how-to-get-unique-values-in-an-array

We could probably stand to have some general helper functions for this kind of stuff somewhere, maybe under 'common'